### PR TITLE
[INTERNAL] specVersion 3.2 docs

### PR DESCRIPTION
**Pull request by d3xter666**
Adds information about specVersion 3.2

**_Note:_** This change shouldn't be merged before:
- https://github.com/SAP/ui5-builder/pull/951
- https://github.com/SAP/ui5-project/pull/673

### CODE REVIEW

File: docs/pages/Configuration.md [hunk 1]
```diff
@@ -750,6 +751,12 @@ Version | UI5 CLI Release
 **1.0** | v1.0.0+
 **0.1** | v0.0.1+
 
+### Specification Version 3.2
+
+**Features:**
+
+- Adds a new bundling mode in sections for providing a dependencies cache: [`builder.bundles.bundleDefinition.sections.mode=depCache`](#properties) configuration for projects of type `module`
```
Comment (matz3) [inline docs/pages/Configuration.md:758]: This is not
> for projects of type `module`

right? Looks like c&p from below

File: docs/pages/Configuration.md [hunk 2]
```diff
@@ -695,7 +695,7 @@ A list of bundle definitions. A `bundleDefinition` contains of the following opt
 - `name`: The module bundle name
 - `defaultFileTypes`: List of default file types which should be included in the bundle. Defaults to: `.js`, `.control.xml`, `.fragment.html`, `.fragment.json`, `.fragment.xml`, `.view.html`, `.view.json` and `.view.xml`
   - `sections`: A list of module bundle definition sections. Each section specifies an embedding technology (see [API-Reference](https://sap.github.io/ui5-tooling/v3/api/module-@ui5_builder_processors_bundlers_moduleBundler.html#~ModuleBundleDefinition)) and lists the resources that should be in- or excluded from the section.
-    - `mode`:  The embedding technology (e.g. provided, raw, preload)
+    - `mode`:  The embedding technology (e.g. provided, raw, preload, depCache)
```
Comment (matz3) [inline docs/pages/Configuration.md:698]: Let's also add the missing `bundleInfo` (see features of specVersion 2.4)
```suggestion
    - `mode`:  The embedding technology (e.g. provided, raw, preload, bundleInfo, depCache)
```

File: docs/pages/Configuration.md (hunk 1 above)
Comment (matz3) [inline docs/pages/Configuration.md:758]:
```suggestion
- Adds a new bundling mode `depCache` for providing a dependencies cache: [`builder.bundles.bundleDefinition.sections.mode=depCache`](#properties)
```

**Review by matz3 — DISMISSED** [review]

File: docs/pages/Configuration.md [hunk 3]
```diff
@@ -750,6 +751,12 @@ Version | UI5 CLI Release
 **1.0** | v1.0.0+
 **0.1** | v0.0.1+
 
+### Specification Version 3.2
+
+**Features:**
+
+- Adds a new bundling mode `depCache` for providing a dependencies cache: [`builder.bundles.bundleDefinition.sections.mode=depCache`](#properties)
+
```
Comment (matz3) [inline docs/pages/Configuration.md:759]: For previous versions we had also a line saying e.g. `Specification Version 3.0 projects are supported by [UI5 CLI](https://github.com/SAP/ui5-cli) v3.0.0 and above.`.

This seems to be missing for 3.1 and 3.2.

Comment (d3xter666) [reply docs/pages/Configuration.md:759]: Done

**Review by matz3 — DISMISSED** [review]

**Comment by RandomByte** [comment]
There's also a bunch of examples where we should update the specVersion to the latest. E.g. https://sap.github.io/ui5-tooling/stable/pages/OpenUI5/#configuration

Search & replace for `specVersion: "3.1"` should yield most of them

**Comment by RandomByte** [comment]
We also need to update the specVersion for `ui5 init`: https://github.com/SAP/ui5-cli/blob/main/lib/init/init.js#L75

**Comment by d3xter666** [comment]
@RandomByte , @matz3 I'll do those changes, yet they still need to after/with the release.

**Review by flovogt — APPROVED** [review]

**Pull request merged**